src/rex.core/src/rex/core/validate.py

```diff
@@ -261,7 +261,10 @@ def include_path(self, node):
                     "expected a file name, but found an empty node",
                     node.start_mark)
         basename = getattr(self.stream, 'name', None)
-        filename = node.value
+        filename = node.value.format(
+            sys_prefix=sys.prefix,
+            cwd=os.getcwd(),
```

Comment:
nit: maybe hoist `os.getcwd()` to the top scope? (this is what we do in other places like `PathVal`.